features/step_definitions/operators.rb

```diff
@@ -160,3 +160,44 @@
   cb_name = 'operator_channel_name' unless cb_name
   cb[cb_name] = cluster_version('version').channel.split('-')[1]
 end
+
+Given /^I wait(?: up to (\d+) seconds)? for operator "([^"]*)" to become conditions: "([^"]*)"$/ do | time, operator, conditions |
+  ensure_admin_tagged
+
+  expected = {}
+  # Parse the conditions to Hash value {Available"=>"True", "Progressing"=>"False", "Degraded"=>"False", "Upgradeable"=>"True"}
+  arr_conditions = conditions.split(/\s+/)    
+  raise "#### Invalid number of conditions!!! (Valid number:1~4)" unless arr_conditions.length <=4
+  if arr_conditions.any? { |x| x.include? "=" }
+    arr_conditions.map{|ar| ar.split('=')}.each {|k, v| expected[k]=v}
+  else
+    # Parse the conditions with "True False ..." input 
+    ["Available", "Progressing", "Degraded", "Upgradeable"][0,arr_conditions.length].zip(arr_conditions).map {|k, v| expected[k]=v}
```

Comment:
@wangke19 actually in Slack I said only wanting to support the short format for below special case:
the condition number is EXACTLY 3 and the conditions EXACTLY are Available, Progressing, Degraded.
So "True False False" is widely understood.
I don't want to support for other cases:
the condition number is 1, 2 or 4, and the conditions are not EXACTLY Available, Progressing, Degraded
So "True True", or "True" are not supported.

Reply:
I get your point. See my previous comment. If we use the keywords with or without `non-` prefix, then we can support in a short format any combination desired. WDYT?

Reply:
For lines 174~175 I changed as following:
```
    # Parse the conditions with "True False ..." input 
    expected_order = {"Available"=>"True", "Progressing"=>"False", "Degraded"=>"False"}
    ["Available", "Progressing", "Degraded"][0,arr_conditions.length].zip(arr_conditions).map {|k, v| expected[k]=v}
    raise "#### Only the brevity conditions order 'True False False' is supported!!!" unless expected == actual_results 
```
There is only one format for qualifying brevity input: True False False